src/ui/common/src/utils/workflows.tsx

```diff
@@ -135,14 +143,22 @@ export type GetWorkflowResponse = {
   workflow_dag_results: WorkflowDagResultSummary[];
 };
 
+export function getSavedObjectIdentifier(savedObject: SavedObject): string {
+  const serviceString: string = savedObject.spec.service;
+  if (Object.values<string>(RelationalDBServices).includes(serviceString)) {
```

Comment:
I followed on of the answers in the top answer here: https://stackoverflow.com/questions/43804805/check-if-value-exists-in-enum-in-typescript

It's a little roundabout imo, but can test to make sure.

Reply:
Looks fine to me. Read through the StackOverflow earlier today and a lot of this stuff boils down to how our code gets transpiled into JS.

Does look a little roundabout, but it is quite type safe in it's check :)